superset-frontend/plugins/legacy-preset-chart-deckgl/src/utilities/Shared_DeckGL.jsx

```diff
@@ -29,6 +29,28 @@ import {
 import { D3_FORMAT_OPTIONS, sharedControls } from '@superset-ui/chart-controls';
 import { columnChoices, PRIMARY_COLOR } from './controls';
 
+let deckgl_tiles;
+
+export const DEFAULT_DECKGL_TILES = [
+  ['mapbox://styles/mapbox/streets-v9', 'Streets'],
+  ['mapbox://styles/mapbox/dark-v9', 'Dark'],
+  ['mapbox://styles/mapbox/light-v9', 'Light'],
+  ['mapbox://styles/mapbox/satellite-streets-v9', 'Satellite Streets'],
+  ['mapbox://styles/mapbox/satellite-v9', 'Satellite'],
+  ['mapbox://styles/mapbox/outdoors-v9', 'Outdoors'],
```

Comment:
Should we be adding OpenStreetmaps (or others) if they're license compatible? The more, the merrier, I figure.

Reply:
OSM tile that I added do not need any key. Instead you add to set up cors options to make it work.
If I underdtand I should replace mapbox tiles by osm ones or mix them in hard coded default tiles ?

Reply:
Hmm... I guess what we want to avoid here is any breaking change. Therefore, we can't shift _existing_ charts to OSM (that's breaking) but we could set the default for NEW charts to OSM, and leave MapBox enabled as an option. I'd also be open to converting the built-in Superset examples to use OSM as well, so things don't look broken when people first install Superset.

Maybe there's enough complexity to warrant a SIP. With that kind of consensus, we would have an idea of the to-do list, and also could get consensus to migrate away from MapBox to OSM if you (and others) think that's a viable option. I'm largely in favor of sensible defaults and configurable options, so long as we don't cause breaking changes, and provide migrations where necessary.